o2a/mappers/null_mapper.py

```diff
@@ -13,19 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Null Mapper - used when there is a need to insert no-op node"""
-from typing import Set
-from xml.etree.ElementTree import Element
+from typing import Set, Tuple, List
 
-from airflow.utils.trigger_rule import TriggerRule
+from o2a.converter.relation import Relation
+from o2a.converter.task import Task
 from o2a.mappers.base_mapper import BaseMapper
 
 
 class NullMapper(BaseMapper):
-    def __init__(self, oozie_node: Element, name: str):
-        BaseMapper.__init__(self, oozie_node=oozie_node, name=name, trigger_rule=TriggerRule.DUMMY)
 
     # pylint: disable=no-self-use
-    def to_tasks_and_relations(self):
+    def to_tasks_and_relations(self) -> Tuple[List[Task], List[Relation]]:
```

Comment:
I think that we should delete this mapper. It is useless. He is useless. When it is used it will always generate an incorrect DAG, because the relationship is not created correctly.

Reply:
Removed in 6cedebe6a5b0c363fcae1bb4af54edf2309d8d7f